### DISCUSSION TITLE
Mirth service manager not displaying

### DISCUSSION THREAD

**Discussion by rutledj** [nextgenhealthcare/connect#5828] [Q&A]
I installed MC on a windows 10 pc in order to play around with using SQL Server instead of Derby. MC runs fine but when I try to start the service manager, I get the UA prompt and select yes but nothing happens. In task manager I see both mcmanager and mcservice running.

Any ideas what might be the issue?

**Comment by tonygermano** [comment]
Did you look for mcmanager running in the system tray after you started it?

**Reply by tonygermano** [reply]
Also, you don't really need mcmanager. mcservice can be stopped and started through the normal windows `services` app. I think the only other thing it does is edit config files for you, but you can do that yourself, too.

**Comment by rutledj** [comment]
It finally showed up there after starting it multiple times. Not sure why it won't show up there the first time you start it.
